src/components/InventoryV2/InventoryBatch.tsx

```diff
@@ -0,0 +1,13 @@
+import TfMain from 'src/components/common/TfMain';
+
+type OriginPage = 'Nursery' | 'Species';
```

Comment:
Is this for tracking from which tab we clicked the batch tab?

Reply:
Yes, the idea is that we can determine from which page we navigated here from and then tell what the path should be, `/inventory/species/X/batch/Y` or `inventory/nursery/Z/batch/Y`. This will allow us to correctly display the right breadcrumb:
<img width="261" alt="image" src="https://github.com/terraware/terraware-web/assets/114949086/5cfcf972-8f69-47cc-9f81-45204c36780d">

That was my idea anyways. Let me know if you think there's a better way! Thanks.

Reply:
Ok, this is for batch details! I thought it was referring to the Batch tab. This looks good as it is!